### ISSUE TITLE
Support "Onliner Spambot" 

### ISSUE THREAD

**Issue by cmlh** [cmlh/Maltego-haveibeenpwned#16]
No `Domain` is specified for the "[Onliner Spambot](https://haveibeenpwned.com/PwnedWebsites#OnlinerSpambot)" breach.

The JSON is reproduced below:

```
     {
        "AddedDate": "2017-08-29T19:25:56Z",
        "BreachDate": "2017-08-28",
        "DataClasses": [
            "Email addresses",
            "Passwords"
        ],
        "Description": "In August 2017, a spambot by the name of <a href=\"https://benkowlab.blogspot.com.au/2017/08/from-onliner-spambot-to-millions-of.html\" target=\"_blank\" rel=\"noopener\">Onliner Spambot was identified by security researcher Benkow mo?u?q</a>. The malicious software contained a server-based component located on an IP address in the Netherlands which exposed a large number of files containing personal information. In total, there were 711 million unique email addresses, many of which were also accompanied by corresponding passwords. A full write-up on what data was found is in the blog post titled <a href=\"https://www.troyhunt.com/inside-the-massive-711-million-record-onliner-spambot-dump\" target=\"_blank\" rel=\"noopener\">Inside the Massive 711 Million Record Onliner Spambot Dump</a>.",
        "Domain": "",
        "IsActive": true,
        "IsFabricated": false,
        "IsRetired": false,
        "IsSensitive": false,
        "IsSpamList": true,
        "IsVerified": true,
        "LogoType": "png",
        "ModifiedDate": "2017-08-29T19:25:56Z",
        "Name": "OnlinerSpambot",
        "PwnCount": 711477622,
        "Title": "Onliner Spambot"
    },
````

**Comment by cmlh** [comment]
Other breach names with no `"Domain": "",` values are:
- `AntiPublic`
- `B2BUSABusinesses`
- `CrimeAgencyVBulletin`
- `DataEnrichment`
- `ExploitIn`
- `VINs`

**Comment by cmlh** [comment]
Fix #16
